### PR TITLE
Add a (skipped) test for creating two comments quickly

### PR DESCRIPTION
**Pull request by danielbachhuber**
We'll need to revisit this after `wp_die()` handling is added.

See #1660, #790

### CODE REVIEW

File: tests/test-rest-comments-controller.php [hunk 1]
```diff
@@ -586,6 +586,45 @@ public function test_create_comment_closed() {
...
+			'content'      => 'Shakes fist at sky',
+		);
+
+		$request = new WP_REST_Request( 'POST', '/wp/v2/comments' );
+		$request->add_header( 'content-type', 'application/json' );
+		$request->set_body( wp_json_encode( $params ) );
+
+		$response = $this->server->dispatch( $request );
+		$response = rest_ensure_response( $response );
+		$this->assertEquals( 400, $response->get_status() );
```
Comment (danielbachhuber) [inline tests/test-rest-comments-controller.php]: @WP-API/amigos does a 400 status seem reasonable here?

**Comment by danielbachhuber** [comment]
Hm. Tests broken by https://core.trac.wordpress.org/changeset/35258/

**Comment by danielbachhuber** [comment]
@WP-API/amigos #reviewmerge

File: tests/test-rest-comments-controller.php (hunk 1 above)
Comment (joehoyle) [inline tests/test-rest-comments-controller.php]: Perhaps http://httpstatusdogs.com/429-too-many-requests "429 Too Many Requests:"

**Comment by rachelbaker** [comment]
Merged #1662

**Pull request merged**